google/cloud/bigtable/tests/admin_integration_test.cc

```diff
@@ -231,6 +233,71 @@ TEST_F(AdminIntegrationTest, CreateListGetDeleteTableTest) {
   EXPECT_EQ(0, table_count);
 }
 
+/// @test Verify that `bigtable::TableAdmin` CheckConsistency works as expected.
+TEST_F(AdminIntegrationTest, CheckConsistencyIntegrationTest) {
+  using GC = bigtable::GcRule;
+  using namespace google::cloud::testing_util::chrono_literals;
+
+  std::string id =
+      "it-" + google::cloud::internal::Sample(
+                  generator_, 8, "abcdefghijklmnopqrstuvwxyz0123456789");
+  auto project_id = bigtable::testing::TableTestEnvironment::project_id();
+
+  auto instance_admin_client = bigtable::CreateDefaultInstanceAdminClient(
+      project_id, bigtable::ClientOptions());
+  bigtable::InstanceAdmin instance_admin(instance_admin_client);
+
+  bigtable::InstanceId instance_id(id);
+  bigtable::DisplayName display_name("Integration Tests " + id);
+
+  // Replication needs at least two clusters
+  auto cluster_config_1 =
+      bigtable::ClusterConfig("us-central1-b", 3, bigtable::ClusterConfig::HDD);
+  auto cluster_config_2 =
+      bigtable::ClusterConfig("us-central1-a", 3, bigtable::ClusterConfig::HDD);
+  bigtable::InstanceConfig config(
+      instance_id, display_name,
+      {{id + "-c1", cluster_config_1}, {id + "-c2", cluster_config_2}});
+
+  auto instance = instance_admin.CreateInstance(config).get();
+
+  // need to create table_admin with longer polling as ConsistencyCheck takes a
+  // little long
+  std::shared_ptr<bigtable::AdminClient> admin_client =
+      bigtable::CreateDefaultAdminClient(
+          bigtable::testing::TableTestEnvironment::project_id(),
+          bigtable::ClientOptions());
+  bigtable::TableAdmin table_admin(
+      admin_client, id,
+      bigtable::GenericPollingPolicy<bigtable::LimitedTimeRetryPolicy,
+                                     bigtable::ExponentialBackoffPolicy>(
+          bigtable::LimitedTimeRetryPolicy(3_min),
+          bigtable::ExponentialBackoffPolicy(10_ms, 10_min)));
+
+  std::string const random_table_id = RandomTableId();
+  google::cloud::bigtable::TableId table_id(random_table_id);
+
+  // create table config
+  bigtable::TableConfig table_config(
+      {{"fam", GC::MaxNumVersions(5)},
+       {"foo", GC::MaxAge(std::chrono::hours(24))}},
+      {"a1000", "a2000", "b3000", "m5000"});
+
+  // create table
+  auto table = table_admin.CreateTable(table_id.get(), table_config);
+
```

Comment:
We should insert at least a little bit of data before getting the consistency token.  Otherwise there is nothing to replicate.

Reply:
ok, i will insert some data. 
Apparently,  it appears that `CheckConsistency` also checks for creation of table across clusters.  `CheckConsistency`  returns success after quite a few calls.

Reply:
Ack.  I did not know that.  In any case, some data would be useful.